lib/oxidized/model/awplus.rb

```diff
@@ -0,0 +1,51 @@
+class Awplus < Oxidized::Model
+
+  #Allied Telesis Alliedware Plus Model#
+
+  prompt /^(\r?[\w.@:\/-]+[#>]\s?)$/
+  comment  '! '
+
+  #This shows the system information at the top and comments it out#
+  cmd 'show system' do |cfg|
+    cfg = cfg.each_line.to_a[0...-2]
+
+
+   # Strip system (up)time/ram/free flash.
+    cfg = cfg.reject { |line| line.match /System Status\s*.*/ }
```

Comment:
Is it really necessary to put all these on one line?

I like the idea of it being separate as the user can then simply remove the lines they're happy to actually see. It just keeps it simple. 

I just change my software on a very regular basis, however most users will not be this case. So that's at least 3 of the 7 lines removed.

Thanks,

Matt